src/Api/ApiClient.php

```diff
@@ -147,14 +163,16 @@ public function put(
 	 * @throws VerificationFailedException
 	 * @throws RequestException
 	 * @throws ApiClientDriverException
+	 * @throws InvalidSignatureException
 	 */
 	public function request(
 		HttpMethod $method,
 		string $url,
 		array $queries = [],
 		array $data = null,
 		SignatureDataFormatter $responseSignatureDataFormatter,
-		\Closure $responseValidityCallback = null
+		\Closure $responseValidityCallback = null,
+		array $extensions
```

Comment:
public proto, kdyby někdo náhodou potřeboval udělat na bránu volání s put (nebo jinou) metodou. Ano, podle současné dokumentace to k ničemu není. Možná můžeme zauvažovat o nějakém nenásilném varování v příští minor verzi. Každopádně teď bych tam tu výchozí hodnotu dal.